packages/expo-av/src/Audio.ts

```diff
@@ -27,8 +27,7 @@ const _isValueValid = (value: any, validValues: any[]): boolean => {
   return validValues.filter(validValue => validValue === value).length > 0;
 };
 
-// Returns array of missing keys in object. Returns an empty array if no missing keys are found.
```

Comment:
I removed this line since this comment was originally written for a function `_findMissingKeys` that has since been deleted.

Reply:
makes sense to me